client/src/components/apps/Apps.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useMemo } from 'react'
+import { useQuery } from '../../api/api'
+import Grid from '../../material/Grid/Grid'
+import Col from '../../material/Grid/Col'
+import AppCards from './appCards/AppCards'
+import Filters from './Filters'
+import Pagination from './Pagination'
+
+const channels = {
+    Stable: 'Stable',
+    Development: 'Development',
+    Canary: 'Canary',
+}
+
+const types = {
```

Comment:
These are already defined in `default.config.js`, under `appTypeToDisplayName`. Might not be the most intuitive place to have it, but we should keep it to one place. Maybe that should also be moved to apiConstants?

Reply:
Yep let's move both constants to `apiConstants`